secretstream/secretstream_new.go

```diff
@@ -0,0 +1,315 @@
+// Copyright 2020 The Go Authors. All rights reserved.
```

Comment:
Can you add an additional note to the top of this file with a link to the open ticket to make this part of core. Such that in the future we can check it and then depend on core instead of this file